tests/test_all.py

```diff
@@ -42,77 +42,87 @@
 """
 
 
-def test_os_release(auto_container):
+def test_os_release(auto_container_per_test):
```

Comment:
Is a conter per test for checking `/etc/os-release` really necessary? After a quick look at https://github.com/dcermak/pytest_container/pull/81 I don't see why this is needed now but I might be missing something. `auto_container` should here also work, no?

Reply:
https://github.com/SUSE/BCI-tests/pull/117#issuecomment-1123548255

Reply:
I see, thanks for the clarification. That was not immediately visible from the two PRs to me.